src/index.html

```diff
@@ -77,27 +77,27 @@ <h2 class="pricing__range">$4<span>/month</span></h2>
 	<!-- end of hero images -->
 
 	<!-- Footer-->
-	<footer class="foot-container">
-		<div class="logo">
+	<footer class="footer">
+		<div class="footer__logo">
 			<img src="./assets/logo.svg" alt="">
 		</div>
-		<div class="center-text">
+		<div class="footer__body">
 			<p>
-				All rights reserved &copy;Equalizer 2022
+				All rights reserved &copy;Equalizer 2022 <br />
```

Comment:
I didn't know i could add br tags to this!

Reply:
a quick way to force the line break !